admin-dev/themes/new-theme/js/pages/order/create/product-renderer.js

```diff
@@ -454,4 +461,22 @@ export default class ProductRenderer {
   _hideNotFound() {
     $(createOrderMap.noProductsFoundWarning).addClass('d-none');
   }
+
+  /**
+   * Shows searching product notice
+   *
+   * @private
+   */
+  _showSearching() {
+    $(createOrderMap.searchingProductsNotice).removeClass('d-none');
+  }
```

Comment:
```suggestion
  _setSearchingState(state) {
    $(createOrderMap.searchingProductsNotice).toggle('d-none', !state);
  }
```

Reply:
I would actually advocate for `showSearching()` and `hideSearching()` because functions with no arguments are easier to use and maintain than functions with N arguments (the higher N is, the more complex it is)

Reply:
It's just depend on you :sweat_smile: 
Ping @NeOMakinG  wdyt?
I really prefer having one method doing
```js
this.display(something);
```

instead of doing

```js
if (something) {
  this.show();
} else {
  this.hide();
}

Reply:
I copied the existent and didn't really think about it, but I actually kind of like the toggle feature better as well But indeed I guess it's a matter of preference, let's leave the final decision to our expert @NeOMakinG ^^

However I wouldn't use `display` because using `display(false)` to hide something feels counter intuitive, Franck's proposal `_setSearchingState` seems more logic, or something like `toggleSearchingNotice` maybe

If we decide to use a toggle though, should it be done the same way or other block toggling (in this class), I also mean the one in this PR https://github.com/PrestaShop/PrestaShop/pull/18334 which are very similar

Reply:
I agree with Pierre on this, I would prefer a toggle function as it decrease complexity to understand it, but that's a matter of preference, except if that's reusable in the future